webrtc-stats.html

```diff
@@ -1515,58 +1217,46 @@ <h2>
                   6.3.1.
                 </p>
               </dd>
-              <dt>
-                <dfn>nackCount</dfn> of type <span class="idlMemberType">unsigned
-                long</span>
```

Comment:
Revive this (nackCount), it was accidentally removed from the definition but not dictionary